Package.swift

```diff
@@ -104,11 +134,11 @@ let package = Package(
     .target(
       name: "GitHub",
       dependencies: [
-        "PointFreePrelude",
+        "DecodableRequest",
+        "EmailAddress",
         .product(name: "Either", package: "swift-prelude"),
+        "FoundationPrelude",
         .product(name: "Logging", package: "swift-log"),
-        .product(name: "Optics", package: "swift-prelude"),
-        .product(name: "Prelude", package: "swift-prelude"),
```

Comment:
by using `DecodableRequest`, `EmailAddress` and `FoundationPrelude` we can stop depending on heavier modules like prelude and optics.